src/shims/ApiCompat.proj

```diff
@@ -77,6 +75,7 @@
 
     <Error Condition="'$(ApiCompatExitCode)' != '0'" Text="ApiCompat failed comparing netstandard.dll to $(TargetGroup)" />
 
+    <!-- TODO: Once the repo targets netstandard2.1, have this instance of APICompat run against NetStandard21RefPath -->
```

Comment:
You'll want an issue to track this I guess.

Reply:
https://github.com/dotnet/corefx/issues/34963